packages/terra-slide-panel/tests/wdio/slide-panel-spec.js

```diff
@@ -53,39 +53,39 @@ Terra.describeViewports('Slide panel', ['large'], () => {
 
   it('Toggles the slide panel and hidden styles', () => {
     browser.url('/raw/tests/cerner-terra-framework-docs/slide-panel/slide-panel-toggle');
-    $('#test-slide [aria-hidden="true"]').waitForExist();
+    $('[aria-label="Main content area"]').waitForExist();
     Terra.validates.element('toggle and hidden styles', { selector: '#root' });
   });
 
   describe('Toggle the slide panel click', () => {
     it('Opens panel and focuses on panel', () => {
       browser.url('/raw/tests/cerner-terra-framework-docs/slide-panel/slide-panel-toggle');
       $('#test-toggle').click();
-      $('#test-slide [aria-hidden="false"]').waitForExist();
+      $('[aria-label="Panel content area"]').waitForExist();
       browser.pause(150);
-      $('#panel-content').isFocused();
+      expect($('[aria-label="Panel content area"]').isFocused()).toBeTruthy();
 
       // On Tab Press focuses on the button inside the panel
       browser.keys(['Tab']);
-      $('#focus-button').isFocused();
+      expect($('#focus-button').isFocused()).toBeTruthy();
 
       Terra.validates.element('panel focused', { selector: '#root' });
     });
     it('Closes panel and focuses on toggle button with keyboard controls', () => {
       browser.url('/raw/tests/cerner-terra-framework-docs/slide-panel/slide-panel-toggle');
 
       browser.keys(['Tab']);
-      $('#test-toggle').isFocused();
+      expect($('#test-toggle').isFocused()).toBeTruthy();
       browser.keys(['Enter']);
-      $('#test-slide [aria-hidden="false"]').waitForExist();
+      $('[aria-label="Panel content area"]').waitForExist();
       browser.pause(150);
 
       browser.keys(['Tab']);
-      $('#focus-button').isFocused();
+      expect($('#focus-button').isFocused()).toBeTruthy();
       browser.keys(['Enter']);
-      $('#test-slide [aria-hidden="true"]').waitForExist();
+      $('[aria-label="Main content area"]').waitForExist();
```

Comment:
I wonder if this one should still be "#test-slide [aria-hidden="true"]" since the main content should already exist (or is this waitForExist even needed since the main content exists?)

Reply:
Good point. I think I need to test that the panel is closed a different way: `[aria-label="Panel content area"][aria-hidden="true"]` I don't want to just rely on the aria-hidden, because the changes in this story allow the main content to also be given aria-hidden="true", which could get ambiguous with how many checks we do throughout the test. Will make that change where needed

Reply:
Gotcha that makes sense, I just realized "test-slide" is the entire main/slide panel component is not just the panel div itself. Checking the panel is closed using aria-hidden = true looks good to me 👍